docs/api/api_blueprint.apib

```diff
@@ -4423,6 +4422,324 @@ Delete a single notification.
           }
         }
 
+# Group Discount Codes
+Discount Codes related to the events.
+
+| Parameter | Description | Type | Required |
+|:----------|-------------|------|----------|
+| `code`  | Discount Code | string | - |
+| `discount-url` | URL of the discount | string | - |
+| `value` | Value of the discount | float | - |
+| `type` | Amount or Percentage | string | - |
+| `is-active` | If the discount code is active | boolean | - |
+| `tickets-number` | Tickets Number for Discount Code | integer (>=0) | - |
+| `min-quantity` | Minimum number of tickets for Discount Code | integer (>=0) | - |
+| `max-quantity` | Maximum number of tickets for Discount Code | integer (>=0) | - |
+| `type` | Amount or Percentage | string | - |
+| `valid-from` | Discount Code valid from | ISO 8601 (tz-aware) | - |
+| `used-for` | Discount Code used for | String | - |
+
+
+## Discount Code Collection [/events/{event_id}/discount-codes{?page%5bsize%5d,page%5bnumber%5d,sort,filter}]
++ Parameters
+    + event_id: 1 (integer) - ID of the specific user in integer
+    + page%5bsize%5d (optional, integer, `10`) - Maximum number of resources in a single paginated response.
+    + page%5bnumber%5d (optional, integer, `2`) - Page number to fetched for the paginated response.
+    + sort (optional, string, `name`) - Sort the resources according to the given attribute in ascending order. Append '-' to sort in descending order.
+    + filter (optional, string, `[{"is_read": "true" }]`) - Filter according to the flask-rest-jsonapi filtering system. Please refer: http://flask-rest-jsonapi.readthedocs.io/en/latest/filtering.html for more.
```

Comment:
@enigmaeth I do not think there is an is_read field in discount-code so this example is not proper. maybe make it `"type": "percentage"` or something similar